cbopensource/connectors/threatconnect/config.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import sys
+
+_logger = logging.getLogger(__name__)
+
+
+class Config(object):
+    # noinspection PyUnusedName
+    feed_name = "threatconnectintegration"
+    # noinspection PyUnusedName
+    display_name = "ThreatConnect"
+    cb_image_path = "/carbonblack.png"
+    integration_image_path = "/threatconnect.png"
+    # noinspection PyUnusedName
+    integration_image_small_path = "/threatconnect-small.png"
+    json_feed_path = "/threatconnect/json"
+    directory = "/usr/share/cb/integrations/cb-threatconnect-connector/content"
+
+    def __init__(self, config_options):
+        self._options = config_options
+        self._errors = []
+        self.debug = self._get_boolean('debug')
+        self.log_level = self._get_string('log_level', "INFO", valid=["DEBUG", "INFO", "WARNING", "ERROR"],
+                                          coerce=True, to_upper=True)
+        self.log_file_size = self._get_int('log_file_size', 10 * 1024 * 1024,
+                                           verify_func=lambda x: x > 0, requirement_message="positive")
+        self.pretty_print_json = self._get_boolean('pretty_print_json')
+        self.multi_core = self._get_boolean('multi_core', True)
+        self.use_feed_stream = self._get_string('feed_save_mode', 'STREAM',
+                                                valid=['STREAM', 'BULK'], to_upper=True) == 'STREAM'
+        self.cache_path = self._get_string('cache_folder',
+                                           "/usr/share/cb/integrations/cb-threatconnect-connector/cache")
```

Comment:
Should this be a class variable instead of a string literal?

Reply:
It's only used in this one spot and it mimics others like server_url.